portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <h1>My Hidden Talents!</h1>
+    <div class="content">
+      <div id="row">
+        <div class="column">
+          <div id="lavendar-background">
+            <h3>Clover-Hunting</h3>
+            <img class="gallery" src="/Images/Clover.png" alt="Clover">
+            <p>One hidden talent that I have is finding four-leaf clovers.
+              Find me a field and I'll find you a clover!
+            </p>
+          </div>
+        </div>
+        <div class="column">
+          <div id="pink-background">
+            <h4>Magic</h4>
+            <img class="gallery" src="/Images/Cards.png" alt="Cards">
+            <p>Another hidden talent I have is magic. I did the Google
+                intern magic workshop and now know exactly one card trick!
+            </p>
+          </div>
+        </div>
+        <div class="column">
+          <div id="blue-background">
+            <h5>Poetry</h5>
+            <img class="gallery" src="/Images/Raven.png" alt="Raven">
+            <p>Finally, I have a talent for memorizing poetry. My sister and
+                I both once memorized the entirety of Edgar Allen Poe's 
+                "The Raven." Definitely a big hit at parties to recite that
```

Comment:
"Definitely a big hit at parties to recite that for fifteen minutes" is not a complete sentence.
```suggestion
                "The Raven." It's a big hit at parties to recite that
```